lib/rubocop/cop/style/it_block_parameter.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Check for replacement with or disallowing of the `it` block parameter.
+      #
+      # It provides three `EnforcedStyle` options:
+      #
+      # 1. `allow_named_parameter` (default) ... Detects only numbered block parameters.
+      # 2. `always` ... Always uses the `it` block parameter.
+      # 3. `disallow` ... Disallows the `it` block parameter.
+      #
+      # A single numbered parameter is detected when `allow_named_parameter` or `always`.
+      #
+      # @example EnforcedStyle: allow_named_parameter (default)
+      #   # bad
+      #   block { do_something(_1) }
+      #
+      #   # good
+      #   block { |it| do_something(it) }
```

Comment:
Same for the others below:

```suggestion
      #   block { do_something(it) }
```